pulp_rpm/app/migrations/0019_migrate_updatecollection_data.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.14 on 2020-08-03 19:21
+
+from django.db import migrations
+
+def rename_update_collections(apps, schema):
+    """Insure every UpdateCollection associated with an Advisory has a unique name"""
+    UpdateCollection = apps.get_model("rpm", "UpdateCollection")
+    UpdateRecord = apps.get_model("rpm", "UpdateRecord")
+    for advisory in UpdateRecord.objects.all():
+        names = {}
+        for collection in advisory.collections.all():
+            if collection.name in names.keys():
+                collection.name = "{}_{}".format(collection.name, names[collection.name])
+                collection.save()
```

Comment:
Shouldn't it be incremented here?
`names[collection.name] += 1`?

Reply:
Also ```defaultdict``` might be useful

Reply:
incr name done - @dalley missed your comment until after I'd already pushed the next round, will take a look tomorrow

Reply:
@dralley I don't think defaultdict helps here, esp since I'm relying on "not in dictionary" to notice the first-time-in case.